clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/security/DcosCredentialMap.groovy

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.clouddriver.dcos.security
+
+class DcosCredentialMap {
```

Comment:
I mean I'm ok with calling it something else as long as it makes more sense than map for this case. Store maybe (since it is used for dumb storage and retrieval)?